x-pack/plugins/lens/public/indexpattern_datasource/_field_item.scss

```diff
@@ -1,47 +1,9 @@
-.lnsFieldItem {
-  @include euiFontSizeS;
-  background: lightOrDarkTheme($euiColorEmptyShade, $euiColorLightestShade);
-  border-radius: $euiBorderRadius;
-  margin-bottom: $euiSizeXS;
-}
-
-.lnsFieldItem__popoverAnchor:hover,
-.lnsFieldItem__popoverAnchor:focus,
-.lnsFieldItem__popoverAnchor:focus-within {
-  @include euiBottomShadowMedium;
-  border-radius: $euiBorderRadius;
-  z-index: 2;
-}
-
 .lnsFieldItem--missing {
   background: lightOrDarkTheme(transparentize($euiColorMediumShade, .9), $euiColorEmptyShade);
   color: $euiColorDarkShade;
 }
```

Comment:
This is no longer working in Lens. It corresponds to the style of the "Empty fields" items which, in this PR, are now getting the white background color instead of gray.

<img width="297" alt="Screen Shot 2020-08-04 at 15 03 39 PM" src="https://user-images.githubusercontent.com/549577/89334020-e2005380-d663-11ea-8d80-7cb00126129d.png">

Reply:
Updated view 
<img width="274" alt="empty" src="https://user-images.githubusercontent.com/4016496/89357545-ab333900-d675-11ea-98f0-8bb059fadabf.png">